internal/cli/users.go

```diff
@@ -22,8 +74,378 @@ func usersCmd(cli *cli) *cobra.Command {
+			}
+
+			a := &management.User{ID: &inputs.ID}
+
+			if err := ansi.Waiting(func() error {
+				var err error
+				a, err = cli.api.User.Read(inputs.ID)
+				return err
+			}); err != nil {
+				return fmt.Errorf("Unable to load user. The Id %v specified doesn't exist", inputs.ID)
+			}
+
+			// get the current connection
+			conn := stringSliceToCommaSeparatedString(cli.getUserConnection(a))
+			a.Connection = auth0.String(conn)
+
+			// parse the connection name to get the requireUsername status
+			u := cli.getConnReqUsername(auth0.StringValue(a.Connection))
+			requireUsername := auth0.BoolValue(u)
+
+			cli.renderer.UserShow(a, requireUsername)
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func deleteUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Delete a user",
+		Long:  "Delete a user.",
+		Example: `auth0 users delete 
+auth0 users delete <id>`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.ID); err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			if !cli.force && canPrompt(cmd) {
+				if confirmed := prompt.Confirm("Are you sure you want to proceed?"); !confirmed {
+					return nil
+				}
+			}
+
+			return ansi.Spinner("Deleting user", func() error {
+				_, err := cli.api.User.Read(inputs.ID)
+
+				if err != nil {
+					return fmt.Errorf("Unable to delete user. The specified Id: %v doesn't exist", inputs.ID)
+				}
+
+				return cli.api.User.Delete(inputs.ID)
+			})
+		},
+	}
+
+	return cmd
+}
+
+func updateUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID         string
+		Email      string
+		Password   string
+		Name       string
+		Connection string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "update",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Update a user",
+		Long:  "Update a user.",
+		Example: `auth0 users update 
+auth0 users update <id> 
+auth0 users update <id> --name John Doe
+auth0 users update -n John Doe --email john.doe@example.com`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.ID); err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			var current *management.User
+
+			if err := ansi.Waiting(func() error {
+				var err error
+				current, err = cli.api.User.Read(inputs.ID)
+				return err
+			}); err != nil {
+				return fmt.Errorf("Unable to load user. The Id %v specified doesn't exist", inputs.ID)
+			}
+			// using getUserConnection to get connection name from user Identities
+			// just using current.connection will return empty
+			conn := stringSliceToCommaSeparatedString(cli.getUserConnection(current))
+			current.Connection = auth0.String(conn)
+
+			if err := userConnection.AskU(cmd, &inputs.Connection, current.Connection); err != nil {
+				return err
+			}
+
```

Comment:
```suggestion
```
Testing this, it doesn't make much sense to ask for connection for update - not something that will be changing much.